index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile-Portofolio</title>
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class= "container">
+        <div class="header">
+            <div><span>Abdi Bekele</span></div>
+            <div class="humber-button"><i class="fa fa-bars"></i></div>
+        </div>
+    
+    <section class= "headline">
+
+        <div class= "heyThere">
+           
+            <h1>Hey There, </h1>
+            <h1>I'm Abdi</h1></div>
+        
+        <div class="career">I am a Software Developer</div>
+        <div class="description"><p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p></div>
+    <div class="links">
+        <ul>
+    <li><img src="./assets/images/github.svg"></li>
+  
+
+    <li><a href="#" class ="link"><i class="fa fa-linkedin-square" ></i></a></li>
+
+    <li><img src="./assets/images/3rdLink.svg"></li>
+
+    <li><img src="./assets/images/twitter4.svg"></li>
+
+    <li><img src="./assets/images/M.svg"></li>
```

Comment:
- [ ] I suggest you use an `a` tag because each icon has a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.
 - like so:
 - ```<li><a><img src="./assets/images/3rdLink.svg"></a></li>```